packages/cli/src/services/util.ts

```diff
@@ -77,8 +77,16 @@ export function pathToPosix (relPath: string): string {
 }
 
 export function splitConfigFilePath (configFile?: string): { configDirectory: string, configFilenames?: string[] } {
+  if (configFile) {
+    const dirname = path.dirname(configFile)
+    const cwd = path.isAbsolute(configFile) ? dirname : path.join(process.cwd(), dirname)
```

Comment:
I think that the code that you have works. I think that [path.resolve](https://nodejs.org/api/path.html#pathresolvepaths) is a bit more concise, though. It will also remove all of the `..` parts of the path.

```suggestion
    const cwd = path.resolve(path.dirname(configFile))
```

Reply:
Good suggestion! Thanks.